db/migrate/20171121113257_migrate_data_from_monitor_link_to_link_history.rb

```diff
@@ -0,0 +1,10 @@
+class MigrateDataFromMonitorLinkToLinkHistory < ActiveRecord::Migration[5.0]
+  def change
+    MonitorLink.all.each do |link|
```

Comment:
Can we use https://apidock.com/rails/ActiveRecord/Batches/find_each to make this more efficient?